.travis.yml

```diff
@@ -29,6 +29,8 @@ before_install:
   # Install dependencies of facter
   - wget https://s3.amazonaws.com/kylo-pl-bucket/yaml-cpp-0.5.1_install.tar.bz2
   - tar xjvf yaml-cpp-0.5.1_install.tar.bz2 --strip 1 -C $USERDIR
+  - wget https://github.com/puppetlabs/leatherman/releases/download/${LEATHERMAN_VERSION}/leatherman.tar.gz
```

Comment:
I tried /latest/ with no luck. We'd have to update this in-sync with the p-a component, at the time that we tag a leatherman release. That's gonna get annoying with more projects using this, unfortunately. No more so than updating submodules would have been, though, and this is easier to see if something is out-of-date at least.